### PR TITLE
EPMRPP-79259 || Asterisks to mandatory fields, EPMRPP-79263 || Unnecessary placeholders remove

### PR DESCRIPTION
**Pull request by tr1ble**

### CODE REVIEW

**Review by Vadim73i — APPROVED** [review]

File: app/src/componentLibrary/fieldText/fieldText.jsx [hunk 1]
```diff
@@ -55,7 +55,12 @@ export const FieldText = ({
 
   return (
     <>
-      {label && <span className={cx(VARIANT, 'label', { disabled })}>{label}</span>}
+      {label && (
+        <span className={cx(VARIANT, 'label', { disabled })}>
+          {label}
+          {isRequired && <span className={cx('asterisk')}>*</span>}
```
Comment (AmsterGet) [inline app/src/componentLibrary/fieldText/fieldText.jsx:61]: Do we need to support case when we have label with asterisk and placeholder with asterisk?
Now it is works as described above.

File: app/src/components/integrations/integrationProviders/rallyIntegration/rallyConnectionFormFields/rallyConnectionFormFields.jsx [hunk 2]
```diff
@@ -81,35 +81,31 @@ export class RallyConnectionFormFields extends Component {
...
           name="project"
           validate={commonValidators.btsProjectId}
           disabled={disabled || editAuthMode}
           className={cx('fields')}
-          placeholder={formatMessage(messages.projectIdLabel)}
           label={formatMessage(messages.projectIdLabel)}
+          isRequired
         >
           <FieldErrorHint provideHint={false}>
             <FieldText defaultWidth={false} maxLength={55} isRequired />
```
Comment (AmsterGet) [inline app/src/components/integrations/integrationProviders/rallyIntegration/rallyConnectionFormFields/rallyConnectionFormFields.jsx:111]: Should it be removed here - ` isRequired` ?

File: app/src/components/integrations/integrationProviders/rallyIntegration/rallyConnectionFormFields/rallyConnectionFormFields.jsx [hunk 3]
```diff
@@ -130,9 +126,10 @@ export class RallyConnectionFormFields extends Component {
           label={formatMessage(messages.accessKeyLabel)}
           disabled={disabled}
           validate={commonValidators.requiredField}
+          isRequired
         >
           <FieldErrorHint provideHint={false}>
-            <FieldTextFlex placeholder={formatMessage(messages.accessKeyLabel)} isRequired />
+            <FieldTextFlex isRequired />
```
Comment (AmsterGet) [inline app/src/components/integrations/integrationProviders/rallyIntegration/rallyConnectionFormFields/rallyConnectionFormFields.jsx:132]: Same here. Please recheck all places.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/componentLibrary/fieldText/fieldText.jsx (hunk 1 above)
Comment (tr1ble) [reply app/src/componentLibrary/fieldText/fieldText.jsx:61]: We need to support placeholder for attributes asterisk

File: app/src/components/integrations/integrationProviders/rallyIntegration/rallyConnectionFormFields/rallyConnectionFormFields.jsx (hunk 2 above)
Comment (tr1ble) [reply app/src/components/integrations/integrationProviders/rallyIntegration/rallyConnectionFormFields/rallyConnectionFormFields.jsx:111]: No, thanks. It doesn't affect anything so I didn't see.

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**